exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/TraceConfigurationTest.java

```diff
@@ -0,0 +1,119 @@
+package com.google.cloud.opentelemetry.trace;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertThrows;
+
+import com.google.auth.Credentials;
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.ServiceOptions;
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import java.util.Collections;
+import java.util.Date;
+import java.time.Duration;
+import java.util.Map;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@link TraceConfiguration}. */
+@RunWith(JUnit4.class)
+public class TraceConfigurationTest {
+
+    private static final Credentials FAKE_CREDENTIALS = GoogleCredentials.newBuilder()
+            .setAccessToken(new AccessToken("fake", new Date(100))).build();
+    private static final String PROJECT_ID = "project";
+    private static final Duration ONE_MINUTE = Duration.ofSeconds(60, 0);
+    private static final Duration NEG_ONE_MINUTE = Duration.ofSeconds(-60, 0);
+
+    @Test
+    public void defaultConfiguration() {
+        TraceConfiguration configuration;
+        try {
+            configuration = TraceConfiguration.builder().build();
+        } catch (Exception e) {
```

Comment:
I am not sure if I understand the testing purpose here. Was this for testing whether an exception will be thrown when project_id is not set? If so, CMIW, I think we can probably just avoid mixing the case here as it seems to be covered by disallowEmptyProjectId()?

Reply:
I'll fix it by setting project ID explicitly.